app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def self.update_count(post)
+    count = post.comments.count
+    post.update(comments_counter: count)
```

Comment:
- [ ] Optional suggestion

I would recommend doing something like this to count the posts.

```
def  self.update_count
  post.increment!(:comments_counter)
end
```

Reply:
Hello @Emmyn5600 
I notice your suggestion. Thanks. It does increase the performance of the app. 
I must say though that in order for it to work, I had to delete the ```self``` prefix.

Again, thank you. 
Best regards!
<img src="https://media1.giphy.com/media/T67JUA2kYbd9S/giphy.gif"/>